<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextPad</class>
 <widget class="QMainWindow" name="TextPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TextPad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/resources/icons/icons8-hand-with-pen-128.png</normaloff>:/icon/resources/icons/icons8-hand-with-pen-128.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="editor">
      <property name="styleSheet">
       <string notr="true">padding: 5px 0 5px 5px;
border: 0 solid transparent;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sansation'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string>start writing. . .</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="editor_statusLbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="noteStateLbl">
      <property name="styleSheet">
       <string notr="true">padding: 0 10px;
border-left: 1px solid #d4d4d4;
background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPage_Setup"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionReplace"/>
    <addaction name="actionGo_To"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionTime_Date"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>For&amp;mat</string>
    </property>
    <addaction name="actionWord_Wrap"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionStatus_Bar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionView_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Textpad"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuEditor">
     <property name="title">
      <string>&amp;Editor</string>
     </property>
     <widget class="QMenu" name="menuTheme">
      <property name="title">
       <string>&amp;Theme</string>
      </property>
      <addaction name="actionTheme_Default"/>
      <addaction name="actionTextpad_Dark"/>
     </widget>
     <addaction name="menuTheme"/>
    </widget>
    <addaction name="menuEditor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">border-bottom: 1px solid #d7d7d7;</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionFind_Next"/>
   <addaction name="separator"/>
   <addaction name="actionView_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>23</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar {
	border-top: 1px solid #d7d7d7;
}</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-add-64.png</normaloff>:/icon/resources/icons/icons8-add-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-opened-folder-64.png</normaloff>:/icon/resources/icons/icons8-opened-folder-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-save-as-64.png</normaloff>:/icon/resources/icons/icons8-save-as-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-send-to-printer-80(1).png</normaloff>:/icon/resources/icons/icons8-send-to-printer-80(1).png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPage_Setup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-page-64.png</normaloff>:/icon/resources/icons/icons8-page-64.png</iconset>
   </property>
   <property name="text">
    <string>Pa&amp;ge Setup</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/unsaved2</normaloff>:/icon/unsaved2</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-undo-80.png</normaloff>:/icon/resources/icons/icons8-undo-80.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-cut-80.png</normaloff>:/icon/resources/icons/icons8-cut-80.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-copy-64.png</normaloff>:/icon/resources/icons/icons8-copy-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-paste-64.png</normaloff>:/icon/resources/icons/icons8-paste-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-delete-64.png</normaloff>:/icon/resources/icons/icons8-delete-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-find-64.png</normaloff>:/icon/resources/icons/icons8-find-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-binoculars-64.png</normaloff>:/icon/resources/icons/icons8-binoculars-64.png</iconset>
   </property>
   <property name="text">
    <string>Find &amp;Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-replace-96.png</normaloff>:/icon/resources/icons/icons8-replace-96.png</iconset>
   </property>
   <property name="text">
    <string>Rep&amp;lace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGo_To">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-linking-80.png</normaloff>:/icon/resources/icons/icons8-linking-80.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Go To</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-checked-64.png</normaloff>:/icon/resources/icons/icons8-checked-64.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionTime_Date">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-calendar-64.png</normaloff>:/icon/resources/icons/icons8-calendar-64.png</iconset>
   </property>
   <property name="text">
    <string>Ti&amp;me/Date</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionWord_Wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-word-wrap-96.png</normaloff>:/icon/resources/icons/icons8-word-wrap-96.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Word Wrap</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-type-80.png</normaloff>:/icon/resources/icons/icons8-type-80.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Font</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Status &amp;Bar</string>
   </property>
  </action>
  <action name="actionView_Help">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-help-80.png</normaloff>:/icon/resources/icons/icons8-help-80.png</iconset>
   </property>
   <property name="text">
    <string>&amp;View Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Textpad">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-about-96.png</normaloff>:/icon/resources/icons/icons8-about-96.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Textpad</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/resources/icons/icons8-redo-80.png</normaloff>:/icon/resources/icons/icons8-redo-80.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionTheme_Default">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Default</string>
   </property>
  </action>
  <action name="actionTextpad_Dark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Textpad Dar&amp;k</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
